### ISSUE TITLE
Newline at the top of the content

### ISSUE THREAD

**Issue by starise** [omnivore-app/obsidian-omnivore#252]
I've found that when importing bookmarks, before the content (after the front matter) there's always a newline at the top. I've solved this annoyance changing the `\n\n` to `\n` from `template.ts`. I'm not sure why the double newline is there.

https://github.com/omnivore-app/obsidian-omnivore/blob/619013b8d659a958d7d4f47cff9172788436276f/src/settings/template.ts#L360

**Comment by SuzanaK** [comment]
Oh it's in the plugin, I thought it was somewhere in my template, it was driving me crazy! 
But if I change it in the template.ts file it will be overwritten with the next plugin update, right?

**Comment by starise** [comment]
> if I change it in the template.ts file it will be overwritten with the next plugin update, right?

Right. I can send a pull request but I was waiting for a feedback that it was not intentional.
